internal/pkg/runtime/launcher/oci/launcher_linux.go

```diff
@@ -320,7 +320,7 @@ func (l *Launcher) finalizeSpec(ctx context.Context, b ocibundle.Bundle, spec *s
 	containerUser := false
 
 	// If the OCI image config specifies a USER we will:
-	//  * When unprivileged - run as that user, via nested subuid/gid mappings (host user -> userns root -> OCI USER)
```

Comment:
Note - this comment wasn't really incorrect... where the parenthesized portion refers to the effective UID at each stage.

However it's easy to read it as if the parenthesized portion describes the ID mapping.. which may have led to this bug.